pkg/k8shandler/deployment.go

```diff
@@ -661,6 +663,12 @@ func (node *deploymentNode) isChanged() bool {
 			changed = true
 		}
 
+		if !comparators.EnvValueEqual(desiredContainer.Env, nodeContainer.Env) {
```

Comment:
There's a comment a few lines prior saying "// Only Image and Resources (CPU & memory) differences trigger rolling restart". Does it change with the addition of the env var comparison?

Reply:
Yes, that will also trigger a rolling restart.

Reply:
Should the comment reflect that? :)